man/docker-build.1.md

```diff
@@ -235,6 +235,13 @@ If you do not provide a version tag then Docker will assign `latest`:
 
 When you list the images, the image above will have the tag `latest`.
 
+Sometimes you want to update the `latest` tag to point to the newly built image
```

Comment:
Your point is buried in the example a bit.  Try this:

-->

You can apply multiple tags to an image. For example, you can apply the `latest`
tag to a newly built image and add another tag that references a specific
version.  For example, to tag an image both as `whenry/fedora-jboss:v2.1` and
`whenry/fedora-jboss:latest`, use the following: 

    $ docker build -t whenry/fedora-jboss:v2.1 -t whenry/fedora-jboss:latest .